tests/E2E/test/campaigns/full/09_customer/customer_settings/1_create_edit_delete_customer.js

```diff
@@ -70,7 +70,10 @@ scenario('Create, Edit, delete "Customer"', () => {
 
   scenario('Check that the customer information is updated in the Front Office', client => {
     test('should go to the Front Office', () => client.switchWindow(1));
-    test('should refresh the page', () => client.refresh());
+    //test('should refresh the page', () => client.refresh());
```

Comment:
Same here. Unused code should be removed, not commented.

Reply:
Removed as well.